app/styles/_header.scss

```diff
@@ -1 +1,88 @@
 // Styles for the header
+.header{
+  margin-top: 2em;
+}
+
+.companyname{
+  background-image: url("../images/companyname.png");
+  background-repeat: no-repeat;
+  background-position: right;
+  width: 8em;
+  height: 2em;
+}
+
+.search{
+ width: 25em;
+ border-color: $grey;
+ background-image: url("../images/search.png");
+ background-repeat: no-repeat;
+ background-position: left;
+}
+
+.counter{
+  background-image: url("../images/counter.png");
+  background-repeat: no-repeat;
+  float:center;
+  width:8em;
+  height:2em;
+}
+
+.menu{
+  margin-top: 1.5em;
+  background-color: $turquoise;
+  text-align: center;
+  font-size: 14px;
+  cursor:pointer;
+
+  a {
+    color:white;
+  }
+}
+
+.colum{
+  border-left: 1px solid white;
+  border-right: 1px solid white;
+  height: 1.5em;
+
+  &:last-child{
+    border-right:none;
+  }
+}
+
+.hero{
+  height: 450px;
+  background-color: $dark;
+  color: white;
+}
+
+.img-hero{
+  position: relative;
+  top: 50%;
+  -ms-transform: translateY(-50%);
+  -webkit-transform: translateY(-50%);
+  transform: translateY(-50%);
+}
```

Comment:
It's very nice that you're using browser prefixes for properties that need it. :) I'll recognise i wasn't expecting it.

However, in this project in the sass task, we're using a gulp plugin to automatically prefix all the properties that need it, take a look https://github.com/diestrin/page-training/blob/master/gulpfile.babel.js#L20

Please remove the prefixes in here to have a cleaner code base. If in any project you're not using a plugin like autoprefixer, you should definitely usse prefixes, but this is not the case ;)